src/desktop/apps/auction/reducers/artworkBrowser.js

```diff
@@ -105,7 +105,8 @@ export default function auctionArtworkFilter(state = initialState, action) {
       )
     }
     case actions.UPDATE_ALL_FETCHED: {
-      if (state.saleArtworks.length === state.total) {
+      const currentPage = state.filterParams.page
+      if (currentPage > 100 || state.saleArtworks.length === state.total) {
```

Comment:
So if I understand correctly this limits us to 100*25 (results/page) - 2,500 artworks/sale? If so, this seems reasonable to me - just want to make sure I understand.

Reply:
Log line in https://artsy.slack.com/archives/CA8SANW3W/p1571265677276700 is what I’m basing the 25 number off of btw

Reply:
It's actually 20 results/page, not 25. The page size can be increased if needed, but the max page of 100 is enforced at the API level, so this enforces that at the client level here too. Always better to be more defensive about that stuff when dealing with an inf. scroll situation.

Reply:
That's set in https://github.com/artsy/force/blob/cc12124d1b1982eac2a9d6b571d798a830ae55ac/src/desktop/apps/auction/reducers/artworkBrowser.js#L20 (also reflected in the log line).